src/CONTRIBUTORS.ts

```diff
@@ -2159,3 +2159,14 @@ export const Swolorno: Contributor = {
     },
   ],
 };
+export const iTitou: Contributor = {
+  nickname: 'iTitou',
+  github: 'titouanc',
+  mains: [
+    {
+      name: 'Fantomeuh',
+      spec: SPECS.RESTORATION_DRUID,
```

Comment:
This is the retail spec. If you want it to be Classic:
`CLASSIC_DRUID_RESTORATION`